hasura/metadata/databases/ibcjuno/tables/public_token.yaml

```
@@ -0,0 +1,18 @@
+table:
+  name: token
+  schema: public
+array_relationships:
+- name: token_units
```

I think we can change this to be simply `units` for easier usage 
```suggestion
- name: units
```